src/RestApiClient.js

```diff
@@ -47,12 +47,16 @@ class RestApiClient {
    * Send a DELETE request to the given path URI.
    *
    * @param  {String} path
+   * @param  {FormData|Object} body
```

Comment:
It doesn't look like this variable is being passed in below, so `body` will be undefined!

Reply:
oh shoot you're right! I'll update in another PR (and the comment below!)